### PR TITLE
Pimo tutorials/04 advanced iv 

### PR DESCRIPTION
**Pull request by jpcbertoldo**
## 📝 Description

add notebook `701e_aupimo_advanced_iv` 

teaches how to load/save from the standard format implemented in https://github.com/jpcbertoldo/aupimo (official aupimo repo)

shows how to perform statistical comparisons between two models ("how significant is the difference between these two models?" or "is model A _consistently_ better than B? (or just by chance?)" 

will be opened after https://github.com/openvinotoolkit/anomalib/pull/2348 is merged

## ✨ Changes

Select what type of change your PR is:

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

For more information about code review checklists, see the [Code Review Checklist](https://github.com/openvinotoolkit/anomalib/blob/main/docs/source/markdown/guides/developer/code_review_checklist.md).

### CODE REVIEW

File: notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb [hunk 1]
```diff
@@ -0,0 +1,936 @@
+{
```
Comment (samet-akcay) [inline notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb:1]: <p>with the&nbsp;<code>anomalib</code>&nbsp;team from Intel's OpenVINO Toolkit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/2352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (jpcbertoldo) [reply notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb]: done

**Comment by jpcbertoldo** [comment]
@samet-akcay can you trigger the CI here please?

**Comment by jpcbertoldo** [comment]
The coverage failure is from the missing gpu issue again

```python
    @staticmethod
    def _choose_gpu_accelerator_backend() -> str:
        if MPSAccelerator.is_available():
            return "mps"
        if CUDAAccelerator.is_available():
            return "cuda"
>       raise MisconfigurationException("No supported gpu backend found!")
E       lightning.fabric.utilities.exceptions.MisconfigurationException: No supported gpu backend found!
```

**Pull request merged**